CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Fixes
+- Fix `Popup` component imports @mnajdova ([#222](https://github.com/stardust-ui/react/pull/222))
+- Fix font size issue in docs @kuzhelov ([#221](https://github.com/stardust-ui/react/pull/221))
```

Comment:
I don't think we should advertise doc changes here, only library changes.  Thoughts on keeping changelog specific to the library itself?